src/js/view/controls/controls.js

```diff
@@ -150,7 +150,12 @@ export default class Controls {
         };
         const settingsMenu = this.settingsMenu = createSettingsMenu(controlbar, visibilityChangeHandler);
         setupSubmenuListeners(settingsMenu, controlbar, model, api);
-        this.div.insertBefore(settingsMenu.element(), controlbar.element());
+
+        if (OS.mobile) {
```

Comment:
Does `OS.mobile` indicate that it's a touch device?

Reply:
It checks wether it is iOS or Android, which what is we currently use as a check in the controls.